### PR TITLE
Social render

### PR DESCRIPTION
**Pull request by bemze**
Fully HTML rendered from JS
Always current year will be up to date

### CODE REVIEW

File: js/components/socialIcons/renderSocialIcons.js [hunk 1]
```diff
@@ -26,6 +31,6 @@ function renderIcons(selector, data) {
                 return false;
             }
         socialsDOM.innerHTML += HTML;
-        
+        console.log(HTML);
```
Comment (pigma15) [inline js/components/socialIcons/renderSocialIcons.js]: console.log paliktas

**Review by pigma15 — APPROVED** [review]
nicen shmeisen :)

File: js/components/socialIcons/renderSocialIcons.js (hunk 1 above)
Comment (bemze) [reply js/components/socialIcons/renderSocialIcons.js]: fixed

**Review by pigma15 — APPROVED** [review]
lgtm

**Review by martynas3113 — APPROVED** [review]
Good job!

**Pull request merged**